kubernetes/samples/charts/voyager/README.md

```diff
@@ -21,6 +26,23 @@ $ helm install appscode/voyager --name voyager-operator --version 7.4.0 \
   --set cloudProvider=baremetal \
   --set apiserver.enableValidatingWebhook=false
 ```
+Wait until Voyager Operator is running.
+```
+$ kubectl -n voyager get all
+NAME                                            READY     STATUS    RESTARTS   AGE
+pod/voyager-voyager-operator-77cbfdcb86-gqwgt   1/1       Running   0          46m
+
+NAME                               TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
+service/voyager-voyager-operator   ClusterIP   10.105.254.144   <none>        443/TCP,56791/TCP   46m
+
+NAME                                       DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/voyager-voyager-operator   1         1         1            1           46m
+
+NAME                                                  DESIRED   CURRENT   READY     AGE
+replicaset.apps/voyager-voyager-operator-77cbfdcb86   1         1         1         46m
+```
+> NOTE: All the generated Kubernetes resources of Voyager operator have names with pattern 'voyager-<releaseName>XXX'. This is the logic controlled by Voyager Helm chart. In our case we use releaseName 'voyager-operator', so all the generated resources have name like 'voyager-voyager-operatorXXX'.
```

Comment:
Edited NOTE.
**NOTE**: All the generated Kubernetes resources of the Voyager operator have names with the pattern `voyager-<releaseName>XXX`. This logic is controlled by the Voyager Helm chart. In our case, we use `releaseName` `voyager-operator`, so all the generated resources have names like `voyager-voyager-operatorXXX`.

Reply:
Thank you, Rose!